docs/dev/discussions.js

```diff
@@ -0,0 +1,428 @@
+/*
```

Comment:
You should remove this file now

Reply:
[0e3a71b](https://github.com/reach-sh/reach-lang/pull/1113/commits/0e3a71bd846654d7f6cb00becb429bb05ef07a2f)